pkgs/applications/office/mmex/default.nix

```diff
@@ -36,6 +47,8 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [
+    appstream # for appstreamcli
+    appstream-glib
```

Comment:
I don't know these packages but these smell like libraries rather than build tools. If these aren't run during the build and are instead intended to be ran by the host platform at runtime, they go in `buildInputs`.

Reply:
(You should set `strictDeps = true;` to ensure these are put into the proper places as that will make the build fail if you do it wrong.)

Reply:
I checked this, they seem to be only used during build. I've also set strictDeps = true;